ocs_ci/deployment/deployment.py

```diff
@@ -855,10 +822,10 @@ def deploy_ocs(self):
             )
 
         # Change registry backend to OCS CEPHFS RWX PVC
-        registry.change_registry_backend_to_ocs()
+        if not config.DEPLOYMENT["external_mode"]:
+            registry.change_registry_backend_to_ocs()
```

Comment:
why are we skipping registry change for external mode?

Reply:
I added this `if` because of the failure in the first PR validation. Should we still be running this? If so it looks like we will need to change the behavior in this function.

Reply:
Yes, we should be running this.
We need to use appropriate value of `sc_name` here [change_registry_backend_to_ocs()](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/registry.py#L28)
Similar to what we are doing in `setup_persistent_monitoring()`

Reply:
Done in latest commit